cernopendata/modules/fixtures/data/articles/news/news.json

```diff
@@ -6,7 +6,8 @@
       "format": "html"
     },
     "collections": [
-      {"experiment": "ATLAS"}
+      {"experiment": "ATLAS"},
```

Comment:
The "experiment" field will be outside of "collections" field.

Reply:
Left it the same way, so when all data model changes are done we can amend (not to change every time)